### PR TITLE
Utilising Github Actions CI

### PR DESCRIPTION
**Pull request by wegotoeleven**
I've made some changes to this repo to take advantage of Github Actions. Essentially, the act of downloading Apple's `config.json` is automated, as is the creation of a PR if the `config.json` has changed. 

For more information about the workflow, please refer to the updated `README.md`.

Feel free to ping me any questions before you accept this merge; it may even be worth merging this with a new branch instead of the main one.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (apizz) [inline .gitignore]: Let's make this `**/.DS_Store` so we exclude all & any instances of .DS_Store files

**Review by apizz — CHANGES_REQUESTED** [review]
Thanks for this!  I'm not familiar with GitHub Actions so curious:
- Where is this ultimately being run?
- Who/what is filing the PR?

**Comment by wegotoeleven** [comment]
- The job is being run on a GitHub hosted "runner", a server hosted by GitHub. These are free, with a limit of [2000 minutes per month](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions). 
- The PR is generated by the runner, [with its own credentials](https://github.com/marketplace/actions/create-pull-request). It will be a pull request against this repository, but could at some point be scripted to create a PR against ProfileManifests. 

If we setup this PR against a new branch, it should run at 00:00 GMT tonight, at which point the PR with the new config will be created.

File: .gitignore (hunk 1 above)
Comment (wegotoeleven) [reply .gitignore]: Done!

**Review by wegotoeleven — COMMENTED** [review]
Changes made to `.gitignore`

**Comment by apizz** [comment]
Let's see how this goes!

**Pull request merged**